packages/web/app/src/components/layouts/organization.tsx

```diff
@@ -50,181 +49,114 @@ export enum Page {
-                  <TabsTrigger variant="menu" value={Page.Members} asChild>
-                    <Link
-                      to="/$organizationSlug/view/members"
-                      params={{ organizationSlug: currentOrganization.slug }}
-                      search={{ page: 'list' }}
-                    >
-                      Members
-                    </Link>
-                  </TabsTrigger>
-                )}
-                <TabsTrigger variant="menu" value={Page.Settings} asChild>
-                  <Link
-                    to="/$organizationSlug/view/settings"
-                    params={{ organizationSlug: currentOrganization.slug }}
-                  >
-                    Settings
-                  </Link>
-                </TabsTrigger>
-                {currentOrganization.viewerCanManageSupportTickets && (
-                  <TabsTrigger variant="menu" value={Page.Support} asChild>
-                    <Link
-                      to="/$organizationSlug/view/support"
-                      params={{ organizationSlug: currentOrganization.slug }}
-                    >
-                      Support
-                    </Link>
-                  </TabsTrigger>
-                )}
-                {getIsStripeEnabled() && currentOrganization.viewerCanDescribeBilling && (
-                  <TabsTrigger variant="menu" value={Page.Subscription} asChild>
-                    <Link
-                      to="/$organizationSlug/view/subscription"
-                      params={{ organizationSlug: currentOrganization.slug }}
-                    >
-                      Subscription
-                    </Link>
-                  </TabsTrigger>
-                )}
-              </TabsList>
-            </Tabs>
-          ) : (
-            <div className="flex flex-row gap-x-8 border-b-2 border-b-transparent px-4 py-3">
-              <div className="h-5 w-12 animate-pulse rounded-full bg-gray-800" />
-              <div className="h-5 w-12 animate-pulse rounded-full bg-gray-800" />
-              <div className="h-5 w-12 animate-pulse rounded-full bg-gray-800" />
-            </div>
-          )}
-          {currentOrganization?.viewerCanCreateProject ? (
-            <>
-              <Button
-                onClick={toggleModalOpen}
-                variant="link"
-                className="text-orange-500"
-                data-cy="new-project-button"
-              >
-                <PlusIcon size={16} className="mr-2" />
-                New project
-              </Button>
-              <CreateProjectModal
-                organizationSlug={props.organizationSlug}
-                isOpen={isModalOpen}
-                toggleModalOpen={toggleModalOpen}
-                // reset the form every time it is closed
-                key={String(isModalOpen)}
-              />
-            </>
-          ) : null}
-        </div>
-      </div>
+      <PrimaryNavigation
```

Comment:
We're no longer prop-drilling the Page enum member through, so this Tabs component was unaware of the active "Tab". We have an existing issue to clean up the inaccessibility of this Tabs usage, so this is a 2-for-1. Lots of interesting bits that were quick-to-build in the new `/navigation` files.